specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,2058 @@
+{

```

Comment:
Why do we need 2 preview API versions (2015 and 2017)?

Reply:
This is a description of the existing API versions

Reply:
Does anyone use the API? As far as I can see, we don't even generate SDKs for the API version.